docs/tools/vdb_table/vendor.schema.json

```diff
@@ -34,7 +34,8 @@
     "sharding": {"allOf": [{"$ref": "#/$defs/featureWithSource"}], "$comment": "Ops | Sharding | The search index is sharded across multiple machines." },
     "doc_size": {"allOf": [{"$ref": "#/$defs/bytesWithSource"}], "$comment": "Ops | Document Size | Maximum size of a stored document." },
     "vector_dims": {"allOf": [{"$ref": "#/$defs/integerWithSource"}], "$comment": "Ops | Vector Dims | Maximum number of vector dimensions." },
-    "int8_quantization": {"allOf": [{"$ref": "#/$defs/featureWithSource"}], "$comment": "Ops | Int8 Quantization | Vectors can be stored in int8 quantized format in the index." },
+    "int8_quantization": {"allOf": [{"$ref": "#/$defs/featureWithSource"}], "$comment": "Ops | Int8 Quantization | Vectors can be stored and indexed in int8 quantized format in the index. Full support requires that the float versions of the vectors need not be stored in the DB at all." },
```

Comment:
Is the computation required to convert float -> int8 (or to binary) always included? I'd assume that partial support is that you can store the quantized vector, but you have to do the transformation yourself vs the full support also does the transformation for you in the database.

But agreed, the question on whether the original float needs to be stored is very important, given that the whole point of quantization is to reduce the RAM/HDD consumption.

Reply:
I think it's fine to leave that to be done outside of the DB. SentenceTransformers provides a straightforward function to do this.